src/styles/index.scss

```diff
@@ -1,3 +1,109 @@
-body {
+@import "./variables";
+@import "./stars.scss";
+
+div,
+a,
+p,
+h2,
+img {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  color: $main-accent;
+}
+
+.card {
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 200px;
+    padding: 32px 16px 16px;
+    background: #fff;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    box-sizing: border-box;
+  }
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__text-reviews {
+    color: $main-accent;
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    color: $secondary;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    text-decoration: none;
+    width: 166px;
```

Comment:
`width` has no effect here

In addition, with such layouts it is better let children fill the width of the parent, to make layout more "fluid"

Reply:
Tests are failed if I don't use "width"

Reply:
I see

It works when removing it with dev tools on your demo deploy, but probably the thing is more complex here